src/modules/AddScoreUI.js

```diff
@@ -1,3 +1,20 @@
+import renderScore from './ScoreUI.js';
+import Scores from './Scores.js';
+
+const onFormSubmit = () => {
+  const addScoreForm = document.querySelector('#form-add-score');
+
+  addScoreForm.addEventListener('submit', async (event) => {
+    event.preventDefault();
```

Comment:
- [ ] [OPTIONAL] Please clear your `input` after submitting the form to show users that you have submitted and the form is empty so you can add another score

Reply:
Hi @hiromataba 
Thank you very much for the invaluable feedback and fast review.

Reply:
Hi @hiromataba 
I have now added the reset form and set focus on name after submit functionalities.

thanks again.